conans/cli/commands/create.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from conans.cli.command import conan_command, Extender, OnceArgument
+
+
+def output_create_cli(info, out):
+    full_reference = info["full_reference"]
+    out.write(full_reference)
+
+
+def output_create_json(info, out):
+    myjson = json.dumps(info, indent=4)
+    out.write(myjson)
+
+
+@conan_command(group="Consumer", formatters={"cli": output_create_cli,
+                                             "json": output_create_json})
+def create(*args, conan_api, parser, **kwargs):
```

Comment:
It still reads weird to me that it is not ``(conan_api, parser, *args, **kwargs)``....